content/en/platform/corda/4.10/enterprise/get-started/get-started-landing.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Get started
+date: '2023-01-27'
+
+menu:
+  corda-enterprise-4-10:
+    identifier: get-started-landing
+    name: "Get started"
+tags:
+- started
+- enterprise
+
+weight: -1
+---
+
```

Comment:
I agree, but there is more content to go under this potentially, so OI want to keep it as a dropdown and not a standalone page. I think it might serve as an index to the other content under it perhaps, like some of the other top-level menu headings. Probably know better once the bulk of the restructuring is done. If we get to end of process and the page still is not needed, we can hardcode the menu heading dropdown in the menu.toml, so that it is not clickable.